wallet-api/src/main/java/org/exoplatform/wallet/rest/WalletAccountREST.java

```diff
@@ -165,7 +165,7 @@ public Response setInitializationStatus(@ApiParam(value = "wallet address", requ
     }
     try {
       accountService.setInitializationStatus(address,
-                                             WalletInitializationState.valueOf(status.toUpperCase()),
```

Comment:
This endpoint is for administrator, normal users could just remove their personal wallets and should not be able to delete wallets of other uses.

Reply:
this endpoint is used only to reject the wallet creation which is not the case anymore: the administration no longer has control over the state of the wallet.

Reply:
done!